src/ToDo/Business/Models/UserContext.cs

```diff
@@ -7,4 +7,5 @@ public record UserContext
 	public string? Email { get; init; }
 
 	public string? AccessToken { get; init; }
+	public byte[]? ProfilePicture { get; init; }
```

Comment:
minor nitpicking: you see the existing code have empty line between members, try to follow the same style as well
gotta keep the coding style consistent